config-lepus.mk

```diff
@@ -11,4 +11,4 @@ JZ_VERSION = 4760
 
 CFLAGS_all := -mips32
 CFLAGS_lepus := -DSDRAM_H5PS1G63EFR_S6C=1 -DDDR_DW32=0
-CFLAGS_gopher2 := -DSDRAM_H2A35121656BB6C=1
+CFLAGS_gopher2 := -DSDRAM_H2A35121656BB6C=1 -DMMC_SD=1
```

Comment:
You don't need `MMC_SD` anymore, do you?

Reply:
I need, see main.c

Reply:
You don't need it in main.c either. MMC_ID2 is defined on all Lepus boards, so booting the external SD should work on all Lepus boards.

Reply:
okay, removed define